android/src/main/java/com/AirMaps/AirMapView.java

```diff
@@ -256,6 +293,65 @@ public void setShowsUserLocation(boolean showUserLocation) {
         map.setMyLocationEnabled(showUserLocation);
     }
 
+    public void setCacheEnabled(boolean cacheEnabled) {
+        this.cacheEnabled = cacheEnabled;
+        this.cacheView();
+    }
+
+    public void enableMapLoading(boolean loadingEnabled) {
+        if (loadingEnabled && !this.isMapLoaded) {
+            this.getMapLoadingLayoutView().setVisibility(View.VISIBLE);
+        }
+    }
+
+    public void setLoadingBackgroundColor(Integer loadingBackgroundColor) {
+        this.loadingBackgroundColor = loadingBackgroundColor;
+
+        if (this.mapLoadingLayout != null) {
+            if (loadingBackgroundColor == null) {
+                this.mapLoadingLayout.setBackgroundColor(Color.WHITE);
+            } else {
+                this.mapLoadingLayout.setBackgroundColor(this.loadingBackgroundColor);
+            }
+        }
+    }
+
+    public void setLoadingIndicatorColor(Integer loadingIndicatorColor) {
+        this.loadingIndicatorColor = loadingIndicatorColor;
+        if (this.mapLoadingProgressBar != null) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                ColorStateList progressTintList = ThemeUtils.getThemeAttrColorStateList(getContext(), android.R.attr.progressTint);
+                ColorStateList secondaryProgressTintList = ThemeUtils.getThemeAttrColorStateList(getContext(), android.R.attr.secondaryProgressTint);
+                ColorStateList indeterminateTintList = ThemeUtils.getThemeAttrColorStateList(getContext(), android.R.attr.indeterminateTint);
+
+                if (loadingIndicatorColor != null) {
+                    progressTintList = ColorStateList.valueOf(loadingIndicatorColor);
+                    secondaryProgressTintList = ColorStateList.valueOf(loadingIndicatorColor);
+                    indeterminateTintList = ColorStateList.valueOf(loadingIndicatorColor);
+                }
+
+                this.mapLoadingProgressBar.setProgressTintList(progressTintList);
+                this.mapLoadingProgressBar.setSecondaryProgressTintList(secondaryProgressTintList);
+                this.mapLoadingProgressBar.setIndeterminateTintList(indeterminateTintList);
+            } else {
+                int color = ThemeUtils.getThemeAttrColor(getContext(), android.R.attr.color);
```

Comment:
@lelandrichardson I was trying to use the default color of progressBar for android. Would including additional dependency on Android Support Library V7 to use as default color be better, or to hard code the color to a fixed default be better?